dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EventDAO.java

```diff
@@ -39,6 +41,27 @@ public long countAllEventsForOrganization(long organizationId) {
         return ((Long)query.getSingleResult()).longValue();
     }
 
+    public List<Event> findEventsByEntryIDs(Set<Long> entryIds, Integer offset, Integer limit) {
+        if (entryIds.isEmpty()) {
+            return Collections.emptyList();
+        }
+        Query<Event> query = namedQuery("io.dockstore.webservice.core.Event.findAllByEntry");
+        query.setParameterList("entryIDs", entryIds).setFirstResult(offset).setMaxResults(limit);
+        return list(query);
+    }
+
+    public List<Event> findEventsByUserId(long userId) {
```

Comment:
These two methods should have sane maxResults and pagination as well.

Reply:
Might be misunderstanding something, why do `findEventsByUserId(long userId)` and `findEventsByEntryId(long entryId)` when versions of both with offset and limits exist? 
Would prefer that we have results limited to something sane in both cases across the board (i.e. delete the non-limited versions) in case someone or something has an epic number of events